### DISCUSSION TITLE
Améliorer les attaques

### DISCUSSION THREAD

**Discussion by Pietagorh** [Crownicles/Crownicles#583] [Idées acceptées / déjà implémentées]
Des compétences et non juste de simples attaques, comme les classes on pourrait les acheter et les utiliser en combat sauf qu'avec ou pourrait stun l'ennemi ou lui mettre un debuff etc.. Et on pourrait faire en sorte que chaque classe équipé à ses propre compétence selon le gameplay.(mais ça se serait une étape 2)

**Comment by BastLast** [comment]
Un rework des combats est prévu à long terme

**Comment by BastLast** [comment]
le rework des combats a eu lieu
